examples/cli-app-with-reqwest/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "cli-app-with-reqwest"
+version = "0.0.0"
+edition = "2021"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+meilisearch-sdk = {path="../.."}
```

Comment:
Here you could the add `no-default-feature` option

Reply:
changed
https://github.com/meilisearch/meilisearch-rust/pull/459/commits/35221ea4d4d3850a5d59a169dd2757441439c803